Source/Core/Core/HW/WiimoteEmu/EmuSubroutines.cpp

```diff
@@ -446,47 +446,7 @@ void Wiimote::DoState(PointerWrap& p)
   p.Do(m_camera_logic.reg_data);
   p.Do(m_ext_logic.reg_data);
   p.Do(m_speaker_logic.reg_data);
-
-  // Do 'm_read_requests' queue
-  {
-    u32 size = 0;
-    if (p.mode == PointerWrap::MODE_READ)
-    {
-      // clear
-      while (!m_read_requests.empty())
-      {
-        delete[] m_read_requests.front().data;
-        m_read_requests.pop();
-      }
-
-      p.Do(size);
-      while (size--)
-      {
-        ReadRequest tmp;
-        p.Do(tmp.address);
-        p.Do(tmp.position);
-        p.Do(tmp.size);
-        tmp.data = new u8[tmp.size];
-        p.DoArray(tmp.data, tmp.size);
-        m_read_requests.push(tmp);
-      }
-    }
-    else
-    {
-      std::queue<ReadRequest> tmp_queue(m_read_requests);
-      size = (u32)(m_read_requests.size());
-      p.Do(size);
-      while (!tmp_queue.empty())
-      {
-        ReadRequest tmp = tmp_queue.front();
-        p.Do(tmp.address);
-        p.Do(tmp.position);
-        p.Do(tmp.size);
-        p.DoArray(tmp.data, tmp.size);
-        tmp_queue.pop();
-      }
-    }
-  }
+  p.Do(m_read_request);
```

Comment:
Does this change the save state layout? If so, please increment the save state version.

Reply:
Yes. Good call.